muxado/src/cancellation_test.rs

```diff
@@ -0,0 +1,50 @@
+use std::{
+    error::Error,
+    time::Duration,
+};
+
+use tokio::{
+    io::AsyncWriteExt,
+    test,
+};
+use tracing::info_span;
+use tracing_test::traced_test;
+
+use crate::*;
+
+#[traced_test]
+#[test]
+async fn test_cancellation() -> Result<(), Box<dyn Error>> {
+    let (left, right) = tokio::io::duplex(256);
+
+    let (dropref, waiter) = awaitdrop::awaitdrop();
+
+    let mut server = info_span!("muxado", side = "server").in_scope(move || {
+        tracing::info!("foo");
```

Comment:
Pointing at this in case you want to remove it or make the `foo` more descriptive.

Reply:
Oh, whoops. Was sanity checking the span context :sweat_smile: